### DISCUSSION TITLE
Bootup Logo Images

### DISCUSSION THREAD

**Discussion by TRaMeLL** [Ralim/IronOS#1119] [Show and tell]
Hi, is there any resource with custom images? I can not find any.
Here are a few of mine:
![5khz_logo](https://user-images.githubusercontent.com/10239461/142725944-b7a4162b-e023-4688-afe7-4f4bf84d0c6f.png)
![sculls_boot_logo](https://user-images.githubusercontent.com/10239461/142725946-aeaac040-82dd-4a50-b052-a806c9c44773.png)
![5khz](https://user-images.githubusercontent.com/10239461/211986122-371211e5-aab9-4eed-8fa1-228552b3b601.png)

**Comment by wiebenor** [comment]
I made a BMP image or two myself, but I'm by no means good at it. One was supposed to be a lightsaber, but I think I don't have that one anymore, and another was supposed to be a profile view of the iron I have which is the TS100, which didn't turn out too bad, that I was able to somehow add some shoddy pixel text to, nothing as good as yours though. Would love to see some more artworks like yours for sure. Would love to have a themed iron myself, say, by using a custom colored plastic casing, and a slogan or something, an example being, a HALO themed green color case, with the startup logo saying "SPARTAN-OS" or something cool like that... Spartans are very versatile, or so legend has it, just like this soldering iron is versatile

**Comment by TRaMeLL** [comment]
Youtube channel logo and Spartan.
![MTW_TS100_BootUp_Logo_by_5kHz](https://user-images.githubusercontent.com/10239461/143389718-4f540682-cd36-433e-b0f8-5e574a5af9ac.png)
![pinecil_spartan_logo](https://user-images.githubusercontent.com/10239461/143389723-02306061-0463-47b4-b39d-0c2bc455fa8a.png)
![WotWU](https://user-images.githubusercontent.com/10239461/143453789-dc82ada0-5a80-4a64-9ee1-c60a9458382b.png)
![pinecil_dizzy_logo](https://user-images.githubusercontent.com/10239461/143467070-6780e5fe-6b72-4049-b8b7-b23b3e6f68dc.png)
![Kilroy_Was_Here](https://user-images.githubusercontent.com/10239461/148887072-3bc7cb0e-7cc1-4bae-ae23-b9f5368f7324.png)

**Reply by wiebenor** [reply]
I don't know what that Russian looking text says, but those, and especially the spartan one look really good. Might have to "borrow" a piece or another of that Spartan one, if it's ok, and try to make a logo out of it, to put on my iron. Tried actually making a helmet on a picture like that, and it looked terrible, yet you managed to not only do 1 Spartan helmet, but 3 DIFFERENT profile angles...

**Reply by TRaMeLL** [reply]
Russian text is "Workers of the world, unite!". It is OK to use graphic elements in your work, of course.
![pinecil_spartan_os1_logo](https://user-images.githubusercontent.com/10239461/143530918-19800509-b85a-4088-8e93-f904d22ff5bd.png)
![pinecil_spartan_os2_logo](https://user-images.githubusercontent.com/10239461/143530919-6a1fe700-cba0-4569-b303-7897857da817.png)
![pinecil_spartan_os3_logo](https://user-images.githubusercontent.com/10239461/143537173-fe972d64-dcd8-40fd-b690-62004014fe84.png)
![pinecil_spartan_os4_logo](https://user-images.githubusercontent.com/10239461/143537179-c2225194-0d99-4d40-8be9-011fce779ecf.png)

**Comment by TRaMeLL** [comment]
Different elements vol.1
![Elements-1](https://user-images.githubusercontent.com/10239461/143681456-a36a1984-7c11-4557-a255-c234b0163aa0.png)
Different elements vol.2
![Elements-2](https://user-images.githubusercontent.com/10239461/144439224-7859bf83-da60-429b-ad25-c3ef8b9b9e0a.png)
It's not a bullet, it's a tip:
![IronOS_Logo-2](https://user-images.githubusercontent.com/10239461/143705555-a3622408-e12a-4729-8d00-1c9345b81bf0.png)
![IronOS_Logo-3](https://user-images.githubusercontent.com/10239461/143721418-1302e971-87ab-4097-836b-6309de4df6da.png)
Mod:
![IronOS_Logo-4](https://user-images.githubusercontent.com/10239461/143734327-ff18c0a4-7432-4f62-9e8c-471ffc2f50d4.png)
![IronOS_Logo-5](https://user-images.githubusercontent.com/10239461/143734529-ff0d690a-ad44-4024-9d7a-304e047b1be3.png)
ASCII:
![IronOS_ASCII](https://user-images.githubusercontent.com/10239461/144585164-4cb75004-fdf5-4a9a-8809-63c451d56864.gif)
[IronOS_ASCII.txt](https://github.com/Ralim/IronOS/files/7648566/IronOS_ASCII.txt)
Omnissiah:
![Omnissiah](https://user-images.githubusercontent.com/10239461/143928553-c40bbdc4-5542-4126-aa46-54638854fc3f.png)
HomeScreen:
![HomeScreen-1](https://user-images.githubusercontent.com/10239461/144439287-d4fd586f-9ff5-4669-a981-d40e5d5e220b.png)
![HomeScreen-2](https://user-images.githubusercontent.com/10239461/144439291-9e660481-6230-4233-8806-c1de9f7a83ac.png)
![TipsDitheringSamples](https://user-images.githubusercontent.com/10239461/147448117-c686f494-730b-4d5c-98f5-36ab7f641a77.png)

**Reply by ghost** [reply]
Really impressive.
Here you have an old discussion showing some examples: #182.
Maybe @Ralim (@discip) would be interested in replacing some icons with yours (#1010, #1078).
I wonder if would be possible for you to share a vectorial image to use, in case, as a repo logo (IronOS with the tip).

Thanks for sharing and congratulations on your work!

**Reply by wiebenor** [reply]
I must agree with federicodilo, the ones you posted before and your new ones are gorgeous, for lack of a more appropriate word... I wonder what application you use to make those, or what process you use to convert them to the right size??? (Of course I'm not saying that you do it the one way or the other)

**Reply by TRaMeLL** [reply]
@federicodilo, I do not mind that my images are used somewhere, for that they are created, if they end up in IronOS, I will be glad. The logo was originally drawn in a resolution of 96x16px, it does not exist in other sizes, it does not exist in vector format either (at least for now).
@wiebenor, Thanks! You can work in this style (Pixel Art) in almost any graphics application, it is more convenient when it has a preview window, such as in "Aseprite". There is no conversion process, I draw pixel by pixel at the original size of 96x16px. As far as I know, it is not possible to convert an image with an acceptable quality to such small sizes.

**Reply by discip** [reply]
Please let me know, if you are ok with this:
https://github.com/Ralim/IronOS/pull/1138

**Reply by TRaMeLL** [reply]
Sure, I am OK with this.

**Reply by discip** [reply]
@TRaMeLL
Any chance the graphics below could be made more recognizable?
[source](https://github.com/Ralim/IronOS/pull/1138#issue-1086296735)
> idle screen: ![idle screen](https://user-images.githubusercontent.com/53649486/147421376-0b0f138b-1fca-4fca-9764-c87dd0dddc6b.png)
> missing tip: ![missing tip](https://user-images.githubusercontent.com/53649486/147421380-0cc0e1a0-6cdf-468f-a8ae-5a31848021be.png)
> 
> reference picture: ![MHP30](https://user-images.githubusercontent.com/53649486/147421481-c66d15f5-0303-40e3-85c9-611017d7bd8a.jpg)

thanks in advance

**Reply by TRaMeLL** [reply]
It is difficult to work from this angle. Maybe under a straight line?
![mhp30-1](https://user-images.githubusercontent.com/10239461/148167857-bb291950-a6b4-46ad-b2ba-034e46a6b880.png)
![mhp30](https://user-images.githubusercontent.com/10239461/148167865-5e398540-da14-4714-8f94-f0428e900a40.png)

**Reply by discip** [reply]
First of all: Thank you for responding.
Since you introduced the 3D style, I would really like to keep it. 😃

I previously did at least 3 designs, but I was not sure wich one is best.
What do you think about the other 2?

1. ![idle screen_MHP30_II](https://user-images.githubusercontent.com/53649486/148316373-8a888a48-3750-4d66-84b8-483dc49d9f3f.png)

2. ![idle screen_MHP30_III](https://user-images.githubusercontent.com/53649486/148316384-bb41be7f-073c-4265-870c-520623603301.png)

Retrospectively I like the 2nd one best, but somehow endet up choosing the one I posted previously.
Do you see a chance to improve them?

thanks in advance
kind regards

**Reply by TRaMeLL** [reply]
12-14 px is not enough space for improvement i am afraid. Both of yours variants are fine, 2nd a little better i guess.
![mhp30-2-Light-1](https://user-images.githubusercontent.com/10239461/148351240-df8e9cbb-b072-4247-a754-abbb87ca949a.png)
![mhp30-2-Light-2](https://user-images.githubusercontent.com/10239461/148351259-1508cd7e-e630-4cb0-ae13-9d4eebdda49b.png)
![mhp30-NoTip-Light-1](https://user-images.githubusercontent.com/10239461/148351269-14004011-c558-4ec0-b066-2906ca06d45a.png)
![mhp30-NoTip-Light-2](https://user-images.githubusercontent.com/10239461/148351279-d1856844-aa99-425d-85df-79af69009a25.png)
![mhp30-NoTip-1](https://user-images.githubusercontent.com/10239461/148353703-3bf56f54-2a40-435f-8832-e45d7595384d.png)
![mhp30-NoTip-2](https://user-images.githubusercontent.com/10239461/148353715-1909c55b-ffc9-445d-a406-4fe421cc09c1.png)
Thin line:
![mhp30-NoTip-TL](https://user-images.githubusercontent.com/10239461/148354417-555e6f42-f9d5-410d-aa32-1cc218f6eb40.png)

**Reply by River-Mochi** [reply]
hello, thank you for making these logos,  are there instructions somewhere on how to get these loaded into Pinecil? thanks

**Reply by Ralim** [reply]
@River-b 
https://github.com/Ralim/IronOS/blob/master/Documentation/Logo.md is the main docs on generating the `.dfu` file to flash from the image

**Comment by River-Mochi** [comment]
thank you @TRaMeLL  I'm using some of your  cute logos now.

**Reply by TRaMeLL** [reply]
You're welcome, I'm glad you liked it! 👍

**Comment by TRaMeLL** [comment]
![cat](https://user-images.githubusercontent.com/10239461/182015910-7bbc9ce5-2eb0-4684-8605-0312a7621d53.png)
![good_luck_symbols](https://user-images.githubusercontent.com/10239461/186382343-34218d77-bc47-4781-b73c-5660a9b90baa.png)

**Reply by River-Mochi** [reply]
these are nice. now we just need a way to get boot logos on new V2 pinecil :)

**Comment by discip** [comment]
@TRaMeLL
Since you are very skilled at creating very clear images in the smallest of spaces, I would be grateful if you had an idea,

1. what symbol could be substituted for the word **`Off`**,
2. how to make it in the existing style at a size of 16x16 ?

Designs I already considered:

I don't know if these are interpreted as **`Off`** by everyone:
1. ![off](https://user-images.githubusercontent.com/53649486/211419359-f0b35283-8c93-4361-a807-bfe223618955.png) This could mean both states (on / off).
2. ![off_b](https://user-images.githubusercontent.com/53649486/211419371-b857f856-089c-4dcf-9e15-45b0a0a546aa.png)
3. ![off_c](https://user-images.githubusercontent.com/53649486/211419379-2d3c026b-b827-4563-927c-6e3bd2628265.png) About this one Ralim said it could be understood as **`unavailable`** instead of **`Off`**.

Also I don't really like the actual design *(flat)*.


Thanks in advance
kind regards

**Reply by TRaMeLL** [reply]
The first symbol is more recognizable, I think.
Something like this maybe:
![PowerOFF-1](https://user-images.githubusercontent.com/10239461/211462424-1ea539c5-2d73-4c62-91f5-5a5804faddec.png)

**Reply by discip** [reply]
@TRaMeLL
Sorry for the late reply.
Thank you for the instant reaction.
Will try your suggestions, most likely there will be slight changes.

**Reply by ghost** [reply]
The black symbol gives the idea of _emptyness_, associated with the concept of switched off. Especially when used in conjunction with its dual (inverted colours).

It must be said, however, that the word 'OFF' is difficult to misunderstand in any case.

**Reply by discip** [reply]
@federicodilo
> The black symbol

Sorry but which one do you mean? 😀
### 1, 2, 3, 4 or 5?

**Reply by ghost** [reply]
I was thinking on wether 2 or 5. Here's a sample how would them be with inverted colours.

![immagine](https://user-images.githubusercontent.com/68966053/214353487-60b40f71-18a9-4a2f-8a23-82b35a5fd5f5.png)

Edit: maybe, according to [Wikipedia](https://en.wikipedia.org/wiki/Power_symbol), you could colour of white, in turns, either the dash or the circle.

Edit 2: Sorry, I would have liked to do some testing but I won't reunite with my TS100 anytime soon. 💔

**Reply by discip** [reply]
1. I'm inclined towards the 2nd variant.

2. However, for what I want to use it for, the **`ON`**-state is not required.
Maybe you could try this [forked build](https://github.com/discip/IronOS/actions/runs/4000198046) to get an Idea.
## But beware! You will have to perform a manual reset as I had to remove the **`invert screen`** & the **`locking mode`** features to make it fit on the **`TS80P`** as well, which was hindered by this (https://github.com/Ralim/IronOS/pull/1536) PR merge.

**Reply by discip** [reply]
Is it dead?

**Reply by ghost** [reply]
Nope, I left it home while completing my studies in the continent. 😁
I shouldn't have it back before 8 months.

**Reply by discip** [reply]
At least there's hope that you'll get it back at all. 😀

**Comment by TRaMeLL** [comment]
Various elements:
![Various-0001](https://github.com/Ralim/IronOS/assets/10239461/efae1005-5756-4549-b1b7-c190477004bb)
